bgs/fedmgr.go

```diff
@@ -294,3 +317,16 @@ func (s *Slurper) GetActiveList() []string {
 
 	return out
 }
+
+func (s *Slurper) KillUpstreamConnection(host string) error {
```

Comment:
So this just kills the existing connection, and prevents a restart unless its re-requested. I'm exposing this as a primitive because theres a number of different ways for it to be used.
The first is for basic load shedding, i can disable new connections then quickly drop a number of PDSs subscriptions, which prevents them from re-connecting unless they have been explicitly added in the database.
The second is that I'll probably put the 'ban' action through this same endpoint, just as an extra flag. This is the high level action that needs to be taken in the case of a ban as well, the other half is just preventing reconnection